modules/social_features/social_search/social_search.install

```diff
@@ -277,3 +277,17 @@ function social_search_update_11401(): string {
   // Output logged messages to related channel of update execution.
   return $updateHelper->logger()->output();
 }
+
+/**
+ * Replace search_api_language with language_with_fallback filter in the search.
+ */
+function social_search_update_11402(): string {
```

Comment:
@volodymyr-sydor Do we need also add some post update hook for  re-index like [here](https://github.com/goalgorilla/cablecar/pull/537/files)?

Reply:
@ribel We could add it into hook_update but seems we don't need it, because after applying changes to the config of indexes, the re-index flag automatically should be set to TRUE, and on the next run of cron content will be re-indexed